app/assets/stylesheets/govuk_publishing_components/components/_cards.scss

```diff
@@ -51,11 +51,15 @@
   // internal margins.
   margin: 0 govuk-spacing(3);
   padding: govuk-spacing(3) 0 govuk-spacing(6) 0;
+}
+
```

Comment:
Could we leave a comment here to explain that the wrapper is there to limit the size of the touch area to differentiate from the one below?

Reply:
i have added a comment to the stylesheet explaining the purpose of the wrapper